vcd/resource_vcd_security_tag.go

```diff
@@ -0,0 +1,113 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdOpenApiSecurityTag() *schema.Resource {
```

Comment:
IMO you can drop the `OpenApi` substring for function names as it doesn't give much value in this context and make it more readable.

```suggestion
func resourceVcdSecurityTag() *schema.Resource {
```

Reply:
Function name changed